lke_clusters.go

```diff
@@ -167,198 +141,138 @@ func (c *Client) ListLKEVersions(ctx context.Context, opts *ListOptions) ([]LKEV
 		return result.([]LKEVersion), nil
 	}
 
-	err = c.listHelper(ctx, &response, opts)
+	response, err := getPaginatedResults[LKEVersion](ctx, c, "lke/versions", opts)
```

Comment:
Can we make `"lke/versions"` be in a variable as the SSoT?

Reply:
Fixed!